lib/Common/Memory/RecyclerPageAllocator.cpp

```diff
@@ -197,7 +197,7 @@ size_t
 RecyclerPageAllocator::GetAllWriteWatchPageCount(DListBase<T> * segmentList)
 {
     size_t totalCount = 0;
-    DListBase<T>::Iterator i(segmentList);
+    _TYPENAME DListBase<T>::Iterator i(segmentList);
```

Comment:
I am following the pattern in DList.h line 456 and 466.
@jianchun  It may be for vs2013 compiler compat?